prime-router/src/main/kotlin/history/azure/HistoryDatabaseAccess.kt

```diff
@@ -148,6 +229,20 @@ abstract class HistoryDatabaseAccess(
     ): Condition {
         var filter = this.organizationFilter(organization, orgService)
 
+        if (receivingOrgSvcStatus != null) {
+            filter = if (receivingOrgSvcStatus == CustomerStatus.ACTIVE) {
```

Comment:
Sorry if my previous comment was confusing, but I meant you could collapse all of this into one statement
```suggestion
           filter = filter.and(
                DSL.jsonbGetAttributeAsText(SETTING.VALUES, "customerStatus")
                    .eq(receivingOrgSvcStatus.name.lowercase())
            )
```

Reply:
Ah, ok didn't know that requirement...so I'm fine leaving this as is, but would argue that the API is now confusing.  I would not expect `TESTING` to come back if I pass `ACTIVE`.  I'm wondering if what you would want to do is support multiple values for this param? i.e.

`?receivingOrgSvcStatus=ACTIVE&receivingOrgSvcStatus=TESTING`

Reply:
If we want to keep it this way, I think we need to capture this behavior via a test.  I think this would be really easy to silently break